tests/10-issuer.js

```diff
@@ -101,50 +126,50 @@ describe('Issue Credential - Data Integrity', function() {
           columnId: name,
           rowId: this.test.title
         };
-        const credential = createValidVC();
-        credential.type = [2, null, {foo: true}, false];
-        const body = {credential};
-        const {result, error} = await issuer.issue({body});
-        shouldThrowInvalidInput({result, error});
+        const {issuer: {id: issuerId}} = issuer;
+        const credential = createValidVc({issuerId});
+        const invalidCredentialTypes = [null, true, 4, []];
+        for(const invalidCredentialType of invalidCredentialTypes) {
+          credential.type = invalidCredentialType;
+          const body = {credential};
+          const {result, error} = await issuer.issue({body});
+          shouldThrowInvalidInput({result, error});
+        }
       });
       it('credential MUST have property "issuer"', async function() {
```

Comment:
@aljones15 DB is failing this test now, instead of throwing error we are getting a 201 response back.

Reply:
yeah this known. We add an issuer if one is not there. The best course of action I can think of is to just to wait and watch, if you're not subscribed to the VC-API, subscribe.